test/fixtures/rules/std_rule_cases/L048.yml

```diff
@@ -1,19 +1,29 @@
 rule: L048
 
-test_pass_1:
+test_pass_simple_select:
   pass_str: "SELECT 'foo'"
 
-test_pass_2:
+test_pass_expression:
   # Test that brackets don't trigger it
   pass_str: "SELECT ('foo' || 'bar') as buzz"
 
-test_fail_simple:
+test_fail_as:
+  fail_str: |
+    SELECT
+        'foo'AS bar
+    FROM foo
+  pass_str: |
```

Comment:
Should this be `fix_str`?

Reply:
Good catch! Fixed. Hopefully the enhancements to #2624 will catch these in future (FYI @juhoautio )